pkg/trace/traceutil/span.go

```diff
@@ -8,23 +8,21 @@ package traceutil
 import "github.com/DataDog/datadog-agent/pkg/trace/pb"
 
 const (
-	// TraceMetricsKey is a tag key which, if set to true,
-	// ensures all statistics are computed for this span.
-	// [FIXME] *not implemented yet*
-	TraceMetricsKey = "datadog.trace_metrics"
-
 	// This is a special metric, it's 1 if the span is top-level, 0 if not.
 	topLevelKey = "_top_level"
+
+	// This is a special meta tag that marks a span for metrics calculation.
+	measuredKey = "_dd.measured"
 )
 
 // HasTopLevel returns true if span is top-level.
 func HasTopLevel(s *pb.Span) bool {
 	return s.Metrics[topLevelKey] == 1
 }
 
-// HasForceMetrics returns true if statistics computation should be forced for this span.
-func HasForceMetrics(s *pb.Span) bool {
-	return s.Meta[TraceMetricsKey] == "true"
+// IsMeasured returns true if a span is marked for measurement.
+func IsMeasured(s *pb.Span) bool {
+	return s.Meta[measuredKey] == "1"
```

Comment:
We usually use the `.Metrics` map for internal markers like this. Let's do the same here, just like we do for top level.